app-containers/incus/autobuild/build

```diff
@@ -0,0 +1,20 @@
+abinfo "Preparing build..."
+mkdir bin
+go mod verify
```

Comment:
Why (you didn't indicate the purpose in `abinfo` either)? If anything, this should be part of Autobuild's template.